modules/index.js

```diff
@@ -0,0 +1,34 @@
+import BookList from './booklist';
+
+const book = new BookList();
+document.getElementById('form').addEventListener('submit', book.add);
+book.loadScreen();
+
+const dt = new Date();
+document.getElementById('date-time').innerHTML = dt;
```

Comment:
- [ ] Here you are getting your date using the vanilla Javascript, however you are  expected to render your date using the `luxon` npm package instead, kindly do that

- [ ] Your page is missing the date below the navigation bar as presented in the images. Kindly fix that.